samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -1,51 +1,239 @@
+/* eslint-disable testing-library/prefer-screen-queries */
 import { expect, test } from '@playwright/test';
+import * as util from './testUtilCommon'
 
 /*
-This test is a simple end-to-end test that verifies the app can be
-loaded, and signed in. The user can send a message to the bot, and expect
-a response.
+This is a test suite to test the functionality of the copilot sample app.
+The Test suite goes through a bunch of tests, opening a new chat session
```

Comment:
nit: you probably don't need this piece and the sentence could just be:
```suggestion
A new chat session is opened for each test so that the chat history is not polluted and the LLM is not confused.
```